src/ExpressionEvaluator/Core/Source/ResultProvider/Helpers/DebuggerPlaceholders.cs

```diff
@@ -0,0 +1,19 @@
+﻿namespace Microsoft.VisualStudio.Debugger.Metadata
+{
+    // Starting in Visual Studio 2015, Update 1, a new api exists to detect if a type is a function pointer.
```

Comment:
@ericfe-ms what is the setup where you are trying to use update1 EE with RTM debugger bits?